sigs.yaml

```diff
@@ -345,12 +346,28 @@ sigs:
       - name: sig-autoscaling-test-failures
         description: Test Failures and Triage
     subprojects:
-    - name: autoscaler
+    - name: scale-client
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/client-go/master/scale/OWNERS # doesn't exist yet
```

Comment:
api machinery owns the level above this. Is it OK for OWNERS files in this list to overlap? I assume it must be. I don't have a strong opinion about who should own this code at the moment. If we had a client-focussed SIG, they would own it, I think, but we don't.